db/fixtures/aws_instance_types.yml

```diff
@@ -0,0 +1,4055 @@
+---
+c1.medium:
```

Comment:
What is this sorted by?  As mentioned earlier, having it sort by name will allow future diffs to be a lot more readable.

Reply:
yes, exactly, it is sorted by name